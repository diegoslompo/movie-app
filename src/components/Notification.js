import React, { Component } from 'react'
import styled from 'styled-components'
import PropTypes from 'prop-types'


class Notification extends Component {

  static propTypes = {
    title: PropTypes.string,
    description: PropTypes.string,
  }

  render () {

    const { title, description } = this.props

    return (
      <Info>
        <Title>{title}</Title>
        <Description>{description}</Description>
      </Info>
    )
  }
} 

const Info = styled.div`
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  height: 120px;
  margin-bottom: 40px;
`;
const Title = styled.h2`
  width: 100%;
  font-size: 45px;
  text-align: center;
  color: #fff;
  margin: 40px 0 0px;
`;
const Description = styled.p`
  width: 100%;
  font-size: 16px;
  text-align: center;
  letter-spacing: 1px;
  color: #fff;
`;


export default Notification
