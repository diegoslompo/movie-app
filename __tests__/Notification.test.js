import React from "react"
import ShallowRenderer from 'react-test-renderer/shallow';
import Notification from '../src/components/Notification'

describe('Testing component Notification', () => { 
    it('should render corretly', () => {
        const renderer = new ShallowRenderer()
        const wrapper = renderer.render(
            <Notification data={{}} />
        )
        expect(wrapper).toMatchSnapshot()
    })
})