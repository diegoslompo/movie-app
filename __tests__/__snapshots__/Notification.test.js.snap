// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component Notification should render corretly 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.h2) />
  <ForwardRef(styled.p) />
</ForwardRef(styled.div)>
`;
